Danger-Swift/DangerFakeSource.swift

```diff
@@ -0,0 +1 @@
+/// Is used for the Package.swift to have a fake Swift source file.
```

Comment:
It looks like `DangerFakeSource` is not defined in this class. Is this something to be implemented in another PR?

Reply:
It's used by implementers like Coyote. I'll update that in the comments

Reply:
Can you explain what you mean by it's used by implementers? Because I only see the comment.

Reply:
It should be clear from the updated comments: https://github.com/WeTransfer/WeTransfer-iOS-CI/pull/42/commits/d5f3c5aefccf17389e5a65389b913f19ac4fc996

Reply:
It does!